packages/react-i18n/src/babel-plugin/test/babel-plugin.test.ts

```diff
@@ -405,6 +531,43 @@ function optionsForFile(filename, withTranslations = false) {
   };
 }
 
+function staticImportOptionsForFile(filename, translationOption) {
+  let dummyPath;
+
+  switch (translationOption) {
+    case 'fallbackOnly':
+      dummyPath = path.join(
```

Comment:
it looks like the import code is longer than the size of the fixtures. inline the fixtures here rather than storing in separate files perhaps?

Reply:
I think they need to separate folders/files so that the plugin detect what files are available in the adjacent `translations` folder. I only need the fixtures to test how the code is generated based on the amount of available locales. Added some symlinks to existing test files.